samples/MobileBuyIntegration/app/src/main/java/com/shopify/checkout_sdk_mobile_buy_integration_sample/logs/LogOverview.kt

```diff
@@ -33,27 +31,23 @@ import androidx.compose.material.TextButton
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.text.style.TextDecoration
-import com.shopify.checkout_sdk_mobile_buy_integration_sample.logs.Logs.DATE_FORMAT
 import com.shopify.checkout_sdk_mobile_buy_integration_sample.logs.Logs.OVERVIEW_FONT_SIZE
-import java.util.Date
 
 @Composable
-fun LogOverviewRow(logDate: LogDate, logMessage: LogMessage, color: Color, onClick: () -> Unit) {
-    Row(Modifier.fillMaxWidth().background(color)) {
-        val formatted = DateFormat.format(DATE_FORMAT, logDate.date)
+fun LogOverviewRow(log: PrettyLog, onClick: () -> Unit, modifier: Modifier) {
+    Row(modifier) {
 
         Text(
-            text = "$formatted",
+            text = log.formattedDate,
             fontSize = OVERVIEW_FONT_SIZE,
-            modifier = Modifier.weight(logDate.weight).fillMaxHeight().align(Alignment.CenterVertically)
+            modifier = Modifier.weight(Logs.DATE_COLUMN_WEIGHT).fillMaxHeight().align(Alignment.CenterVertically)
```

Comment:
I'm making a tradeoff here of allowing this component to control weight by itself rather than receiving it as param. The benefit of that approach is that we can now get rid of the two data classes we created to pass the weight.